packages/perseus/src/widgets/interactive-graphs/axis-ticks.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from "react";
+
+import {useTransform} from "./graphs/use-transform";
+
+import type {vec} from "mafs";
+
+const tickSize = 10;
+
+const tickStyle: React.CSSProperties = {
+    stroke: "black",
+    strokeWidth: 1,
+};
+
+const YGridTick = ({y}: {y: number}) => {
+    const startingPoint: vec.Vector2 = [0, y];
+    let [startPtPx] = useTransform(startingPoint);
+
+    const endPtPx = [startPtPx[0] + tickSize / 2, startPtPx[1]];
+    startPtPx = [startPtPx[0] - tickSize / 2, startPtPx[1]];
+
+    return (
+        <g className="y-axis-ticks" tabIndex={0} style={{} as any}>
```

Comment:
What does the `tabIndex` do for us on this element? I'm wondering what the use case is for it.

Reply:
I believe this snuck in here as we copied the "g" element from elsewhere! Currently getting rid of it :)